packages/staking/src/governance/api.ts

```diff
@@ -0,0 +1,62 @@
+import {AxiosInstance} from 'axios'
+import {axiosClient, GOVERNANCE_ENDPOINTS} from './config'
+
+export type GovernanceApi = {
+  getDRepById: (drepId: string) => Promise<{txId: string}>
```

Comment:
what is the drepId? (drep-key/tx-hash)? is there a similar API for the CIP95? if there is let's try to make as similar as possible.